packages/user-menu/src/UserMenu.js

```diff
@@ -1,48 +1,108 @@
 import React, { Component } from 'react';
-import FlyoutButton from '../../flyout-button/src/FlyoutButton';
-import Button from '../../button/src/Button';
 
-type User = {
-  firstName?: string,
-  lastName?: string
-}
-type LogoutFunction = () => void;
+import Avatar from '../../avatar/src/Avatar';
+import Button from '../../button/src/Button';
+import Flyout from '../../flyout/src/Flyout';
 
 type Props = {
-  user?: User,
-  onLogout?: LogoutFunction,
-  isLoading?: boolean,
-  className?: string,
-  style?: object,
-  children?: any
+  children?: any,
+  /** Login URL */
+  loginUrl: string,
+  /** Is the user logged in? */
+  loggedIn: boolean,
+  /** First name */
+  firstName: string,
+  /** Last name */
+  lastName: string,
+  /** Avatar URL */
+  avatarUrl?: string,
+  /** Logout URL */
+  logoutUrl: string,
 }
-const UserMenu = (props: Props) => {
-  const { user, onLogout, isLoading, children, className, style } = props;
-  const {
-    firstName = '',
-    lastName = '',
-  } = user;
-
-  const name = `${firstName} ${lastName}`;
-  return (
-      <FlyoutButton
-      flyoutDirection="right"
-      iconLeft="user"
-      flyoutSize="small"
-      label={name}
-      style={style}
-      className={className}
-    >
-      {children}
-      <Button
-        iconLeft="power-off"
-        type="danger"
-        block
-        onClick={onLogout}>
-        Uitloggen
-        </Button>
-      </FlyoutButton>
-  )
+
+class UserMenu extends Component<Props> {
+  renderAvatar() {
+    return (
+      <Avatar
+        image={ this.props.avatarUrl || 'https://www.antwerpen.be/assets/overzicht/gfx/no-thumbnail.svg' }
```

Comment:
I don't think pointing to an image is a good idea. Instead, you should use the `fa-user` icon, like demoed here: https://a-ui.github.io/core_branding_scss/#atoms-avatar